src/ansys/dpf/core/__init__.py

```diff
@@ -1,7 +1,30 @@
 import os
 
-from ansys.dpf.core._version import __version__
+# Get version information
+try:
+    import importlib.metadata as importlib_metadata
+except ModuleNotFoundError:
+    import importlib_metadata
+
+__version__ = importlib_metadata.version("ansys-dpf-core")
```

Comment:
Use `__name__` here. It avoids hardcoding the namespace of the package:

```suggestion
__version__ = importlib_metadata.version(__name__.replace(".", "-"))
```

Reply:
Yes, I had this before. However, I was getting the following error in the ``Test docstrings`` step: "importlib.metadata.PackageNotFoundError: core".

With @germa89 we realized that this was fixed by hard-coding the name. I will try to investigate this a bit more.